content/installation/node/NodeSupportedTechnologies.md

```diff
@@ -4,30 +4,44 @@ description: "List of supported technologies"
+
+
+## Third-Party Module Support
+
+* [express-session](https://github.com/expressjs/session)
+* [MongoDB](https://docs.mongodb.org/ecosystem/drivers/node-js/)
+* [MySQL](https://www.npmjs.com/package/mysql)
+* [Mongoose](https://mongoosejs.com)
+* [postgres](https://www.npmjs.com/package/pg)
+* [Sequelize](http://docs.sequelizejs.com/)
 
 Contrast doesn't guarantee support for old or deprecated versions of third-party modules.
 
 ## OS Support
 
-The agent runs entirely in the Node.js application layer with no *C* dependencies. As a result, it works on **all of the same operating systems as Node.js**, including Linux, Windows, macOS and Unix.
+The agent runs entirely in the Node.js application layer with no *C* dependencies. As a result, it works on **the same operating systems as Node.js**, including Linux, Windows, macOS and Unix.
 
 ## Testing Environments
 
-When changes are made, Contrast runs a battery of automated tests to ensure that it detects findings in supported technologies across all supported versions of Node. These automated tests exercise the Contrast agent against Contrast's [Node Test Bench](https://github.com/Contrast-Security-OSS/NodeTestBench) application. This application is updated as more third-party library support is added to the agent. 
+When changes are made, Contrast runs a battery of automated tests to ensure that it detects findings in supported technologies across all supported versions of Node. This includes tests that exercise the agent against Contrast's
+[Node Test Bench](https://github.com/Contrast-Security-OSS/NodeTestBench), [Hapi 16 Test Bench](https://github.com/Contrast-Security-OSS/Hapi16TestBench), and [Hapi 17 Test Bench](https://github.com/Contrast-Security-OSS/HapiTestBench) applications. Each of these applications is updated as Contrast adds more third-party library support to the agent.
 
-> If you want to add test cases, let Contrast know by submitting a pull request. 
+If you want to add test cases, let Contrast know by submitting a pull request.
 
-## Additional Technology 
+## Additional Technology
 
-Contrast runs with a range third-party technology, all of which isn't listed here. If you'd like to confirm that Contrast supports your preferred technology, or you'd like to make a case for Contrast to support it, [let the Contrast team know.](mailto:bugs@contrastsecurity.com)
+Contrast runs with a range of third-party technology, all of which isn't listed here. If you want to confirm that Contrast supports your preferred technology, or you'd like to make a case for Contrast to support it, [let Contrast know.](mailto:bugs@contrastsecurity.com)
```

Comment:
`all of which isn't listed here` -> `not all of which are listed here`

Reply:
@Avuxo I kept "isn't" for the singular "range". But, if you think it sounds awkward, we can rephrase! 

I'm also thinking that we should say "some of which..." since we do mention some options.

Reply:
I feel like it sounds kinda clunky even with the singular `technology`. If nobody else is bothered though, it's fine.